app/src/main/res/layout-sw600dp-port/onboarding_fragment.xml

```diff
@@ -67,6 +67,7 @@
         android:layout_marginTop="12dp"
         android:layout_marginEnd="32dp"
         android:layout_marginBottom="12dp"
+        android:contentDescription="@string/next_arrow"
```

Comment:
apply code reformat so that all these items get arranged in the preferred format. 
You can find `Reformat file` option in the code menu in the Android Studio. 

Apply the same for all the XML files in this pr.

Reply:
@anandwana001 

![image](https://user-images.githubusercontent.com/58650668/114342161-f98bc680-9b78-11eb-9b2a-abf5e924c2e2.png)
I tried running Running ```Reformat file``` in all the four XML layouts but I got the same message for each of them. **"No lines changed: content is already properly formatted."** Is it fine or have I done something wrong ?